test/helpers/handlers/IonHandlerForkBase.sol

```diff
@@ -58,8 +60,12 @@ abstract contract IonHandler_ForkBase is IonPoolSharedSetup {
         }
     }
 
+    /**
+     * The asset that the lenders deposit into the IonPool. By default, the test
+     * assumes the base asset is WETH.
+     */
     function _getUnderlying() internal pure virtual override returns (address) {
-        return address(weth);
+        return address(WETH_ADDRESS);
```

Comment:
This is the mainnet WETH `0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee`, not the Base WETH,

Is this intended?